src/config.ts

```diff
@@ -296,12 +317,28 @@ export const samlConfig: SamlConfig = {
 };
 
 // Redirection urls
-export const clientProfileRedirectionUrl =
-  process.env.CLIENT_REDIRECTION_URL || "/profile.html?token={token}";
-
-if (!clientProfileRedirectionUrl.includes("{token}")) {
-  log.error("CLIENT_REDIRECTION_URL must contains a {token} placeholder");
-}
+export const clientProfileRedirectionUrl = pipe(
+  // this env variable is required and must be an absolute URL
+  process.env.CLIENT_PROFILE_REDIRECTION_URL,
+  NonEmptyString.decode,
+  E.mapLeft((err) => {
+    throw new Error(
+      `CLIENT_PROFILE_REDIRECTION_URL env variable is required | ${readableReportSimplified(
+        err
+      )}`
+    );
+  }),
+  E.chain(
+    E.fromPredicate(
+      (url) => url.includes("{token}"),
+      () => undefined
+    )
+  ),
+  E.getOrElseW(() => {
+    log.error("CLIENT_REDIRECTION_URL must contain a {token} placeholder");
```

Comment:
```suggestion
    log.error("CLIENT_PROFILE_REDIRECTION_URL must contain a {token} placeholder");
```

Reply:
solved in https://github.com/pagopa/io-backend/pull/1012/commits/2f27b7e1455672699d54b3b8e5b38750db8bb08f